docs/data/pages.ts

```diff
@@ -251,8 +251,15 @@ const pages: MuiPage[] = [
         ],
       },
       { pathname: '/x/react-date-pickers/fields', title: 'Field components', newFeature: true },
-      { pathname: '/x/react-date-pickers/custom-components', title: 'Custom subcomponents' },
-      { pathname: '/x/react-date-pickers/custom-layout', title: 'Custom layout' },
+      {
+        pathname: '/x/react-date-pickers/visual-customization',
+        title: 'Visual customization',
```

Comment:
I'm not sure about this title. 🤔 
The possible customizations do not have to be visual 🤔 
Maybe just go with?
```suggestion
        title: 'Customization',
```

Reply:
Customization can also be about the behavior

Reply:
Fair point. Most of the listed examples in the those pages are visual customizations. 👌